config/develop/bridgeserver2.yaml

```diff
@@ -0,0 +1,17 @@
+template_path: templates/bridgeserver2.yaml
+stack_name: bridgeserver2-develop
+parameters:
+  AppHealthcheckUrl: 'HTTP:80/'
+  AwsAutoScalingMaxSize: '1'
+  AwsAutoScalingMinSize: '1'
+  AwsDefaultVpcId: vpc-9c70bbf9
```

Comment:
Created a new VPC using the linked template and used that VPC instead.

Reply:
I would recommend the creation and deployment of the VPC be done in a separate PR.  Also instead of manually creating this VPC we should add it to bridge-infra repo so we can have travis manage it for us.

The files to be added would be similar to the one on sandbox-infra repo:

create vpc: 
https://github.com/Sage-Bionetworks/sandbox-infra/blob/master/config/prod/sandcastlevpc.yaml

setup peering so private subnets are accessible from Sage VPN:
https://github.com/Sage-Bionetworks/sandbox-infra/blob/master/config/prod/peer-vpn-sandcastlevpc.yaml

Reply:
You have permission to fork now and make a peering PR on admincentral-infra now.  Add a config/prod/peering-bridgeserver2-vpc.yaml file with the following content:

```
template_path: templates/VPCPeer.yaml
stack_name: peering-bridgeserver2-vpc
parameters:
  PeerVPC: vpc-XXXXXXXXXXX
  PeerVPCOwner: !ssm /infra/BridgeProdAwsAccountId
  PeerVPCCIDR: 172.XX.0.0/16
  PeerRoleName: bridge-VPCPeeringAuthorizerRole-1XKEYVXHDOYI5
```

Reply:
Created pull request for VPC peering https://github.com/Sage-Bionetworks/admincentral-infra/pull/59

I wasn't able to test or execute this since it appears that this CloudFormation template is managed in a separate AWS account.